src/components/pager/__internal__/pager.component.js

```diff
@@ -22,6 +22,7 @@ const Pager = ({
   onFirst,
   onPrevious,
   onLast,
+  variant,
```

Comment:
I think you're better off putting the default value here rather than on `StyledPagerContainer` because then it'll show the default on the prop table on the docs

Reply:
Currently looks like this, with a `-` in the default column: 
![image](https://user-images.githubusercontent.com/14963680/107670991-3af01a80-6c8b-11eb-972a-da0327fe04ba.png)